gitian-building.md

```diff
@@ -106,6 +106,8 @@ bin/make-base-vm --suite bionic --arch amd64 --docker
 popd
 ```
 
+If you encounter an error about connection timeouts on 172.17.0.1:3142, you probably have blocked port 3142 for apt-cacher-ng.
```

Comment:
Thanks! Might be good to remove the IP address and just mention "port 3142", and add instructions to fix or the link to the tuturial.

Reply:
Thank you for the review. I will add more/better instructions.